### PR TITLE
removed unused function and unneeded gitignore portions

### PR DESCRIPTION
**Pull request by aberke**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,22 +21,6 @@
        "url":"http://arielnoyman.com"
       }
     </script>
-=======
```
Comment (RELNO) [inline index.html]: this is fixed in https://github.com/CityScope/CS_cityscopeJS/commit/dbae0c6cab4b6d350e33c102396729a6be7bc9f8 so could be removed from PR

**Comment by RELNO** [comment]
@aberke can you explain why UpdateinfoDiv() is not used?

**Comment by aberke** [comment]
updateInfoDIV at the bottom of the file seems to be used instead, no? 
https://github.com/CityScope/CS_cityscopeJS/pull/20/commits/d71b3d15c706115fb4e54e632b0ccd6d8dfc3716#diff-4aa09135d3201fd450d1dac5994178e1R279

File: index.html (hunk 1 above)
Comment (aberke) [reply index.html]: done

**Comment by RELNO** [comment]
Good catch, thanks!

**Pull request merged**